.drone.yml

```diff
@@ -16356,10 +16354,12 @@ steps:
   - echo "$RELEASES_CERT" | base64 -d > "$RELCLI_CERT"
   - echo "$RELEASES_KEY" | base64 -d > "$RELCLI_KEY"
   - trap "rm -rf /tmpfs/creds" EXIT
-  - |-
-    docker run -i -v /tmpfs/creds:/tmpfs/creds \
-      -e DRONE_REPO -e DRONE_TAG -e RELCLI_BASE_URL -e RELCLI_CERT -e RELCLI_KEY \
-      $RELCLI_IMAGE auto_publish -f -v 6
+  - docker run -i -v /tmpfs/creds:/tmpfs/creds -e DRONE_REPO -e DRONE_TAG -e RELCLI_BASE_URL
+    -e RELCLI_CERT -e RELCLI_KEY $RELCLI_IMAGE auto_publish -f -v 6 || true
+  - docker run -i -v /tmpfs/creds:/tmpfs/creds -e DRONE_REPO -e DRONE_TAG -e RELCLI_BASE_URL
+    -e RELCLI_CERT -e RELCLI_KEY $RELCLI_IMAGE auto_publish -f -v 6 || true
+  - docker run -i -v /tmpfs/creds:/tmpfs/creds -e DRONE_REPO -e DRONE_TAG -e RELCLI_BASE_URL
+    -e RELCLI_CERT -e RELCLI_KEY $RELCLI_IMAGE auto_publish -f -v 6
```

Comment:
Wouldn't this just always call the command 3 times? Can we instead wrap this in a "for" loop with a few iterations and abort it on success? Also, why 3? I think @logand22 said for him it worked on like 25th attempt, can we just make it retry 50 times or something?

Reply:
It worked on second attempt for me. I mentioned 25 because I called the command many times after the first success to make sure it never occurred after the first success.

Reply:
I think all it needs is `|| \` on the end instead of `|| true`. Then it is one long pipeline that stops when one of the commands succeeds and fails when all fail.

Reply:
But if we do want to do 50 times or whatever, then a loop is a lot better. It shouldn't be that messy - let me knock something up.